Realm/ObjectStore/list.hpp

```diff
@@ -106,6 +106,8 @@ class List {
         size_t valid_count;
     };
 
+    LinkViewRef get_linkview() const { return m_link_view; }
```

Comment:
If you're referring to `Results::get_linkview`, it's not used after realm/realm-object-store@ef1c6ddc63d71a21c87434c056c6097ba8644abb, and should likely have been removed.